src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -199,7 +199,7 @@ public static string SymbolsFont
 			/// The default font family for text when a font isn't explicitly specified (e.g. for a TextBlock)
 			/// This is often needs to be set by users on Linux, where Microsoft's Segoe UI isn't present
 			/// </summary>
-			public static string DefaultTextFontFamily { get; set; } = "Segoe UI";
+			public static string DefaultTextFontFamily { get; set; } = "ms-appx:///Uno.Fonts.OpenSans/Fonts/OpenSans-Regular-VariableFont_wdth,wght.ttf";
```

Comment:
If we don't have that already, let's make sure to document that property in the fonts docs.